cli/src/testenv_utils.rs

```diff
@@ -1,5 +1,3 @@
-// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
```

Comment:
Shouldn't have the CI caught that ?

Reply:
The CI as run the license check:

```
 license_headers..........................................................Passed
- hook id: license_headers
- duration: 0.08s
```

But something seems wrong with the script :/

Reply:
Waiting for #8060 to run check